presto-product-tests/src/main/resources/sql-tests/testcases/connectors/mysql/describe_table.sql

```diff
@@ -1,6 +1,6 @@
 -- database: presto; groups: mysql_connector; queryType: SELECT; tables: mysql.workers_jdbc
 --!
-describe mysql.test.workers_jdbc
+describe mysql.swarm.workers_jdbc
```

Comment:
I think test was better. Once teradata support for presto is official I don't think using name swarm have any sense.